B/BerkeleyDB/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "BerkeleyDB"
+version = v"18.1.40"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.oracle.com/berkeley-db/db-$(version).tar.gz", "0cecb2ef0c67b166de93732769abdeba0555086d51de1090df325e18ee8da9c8")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/db-*/docs
+# see https://stackoverflow.com/questions/64707079/berkeley-db-make-install-fails-on-linux 
+mkdir bdb-sql
+mkdir gsg_db_server
+cd ../build_unix
+../dist/configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(platform -> !(Sys.iswindows(platform)), supported_platforms(exclude=[Platform("powerpc64le", "linux")]))
```

Comment:
Not sure why mixing two different filtering mechanisms
```suggestion
platforms = filter!(p -> !Sys.iswindows(p) && arch(p) != "powerpc64le", supported_platforms())
```

Reply:
For PowerPC, did you try https://docs.binarybuilder.org/stable/troubleshooting/#Shared-library-not-built?

Reply:
Tried with the most recent commit, didn't seem to help. Maybe autoreconf is hitting a bug?

Reply:
Nah, just people thinking that not shipping M4 macros is a good idea